Vagrantfile

```diff
@@ -1,43 +1,50 @@
 # vim: set filetype=ruby:
 
-FLAVOUR = ENV.fetch('FLAVOUR', 'dev')
-unless %w(dev acceptance production).include?(FLAVOUR)
-  $stderr.puts <<-EOF
-    Unknown flavour #{FLAVOUR.inspect}
-
-    Environment variable FLAVOUR must be one of 'dev', 'acceptance' or
-    'production'.  Defaults to 'dev'.
-  EOF
-  exit 1
-end
+Vagrant.configure("2") do |config|
+  # Attempt to load vagrant-env
+  begin
+    config.env.enable
+  rescue Exception
+    $stderr.puts "Failed to enable vagrant-env, continuing..."
+  end
 
-CODE_PATH = ENV['FLIGHT_CODE'] || "#{ENV['HOME']}/code"
-if FLAVOUR == 'dev' && !File.directory?(File.expand_path(CODE_PATH))
-  $stderr.puts <<-EOF
-  Code path is not set.
-  
-  Either set the environment variable FLIGHT_CODE or create a symlink at
-  $HOME/code pointing to the directory containing your OpenFlight github
-  repositories.
-  EOF
-  exit 1
-end
+  FLAVOUR = ENV.fetch('FLAVOUR', 'dev')
+  unless %w(dev acceptance production).include?(FLAVOUR)
+    $stderr.puts <<-EOF
+      Unknown flavour #{FLAVOUR.inspect}
```

Comment:
What was the reason for moving these back inside the `Vagrant.configure` block?

Reply:
The FLAVOUR is set via an env var which could be modified by the .env file. So it should be set after .env is loaded.